### PR TITLE
Load legacy resources like comments.js

### PR DESCRIPTION
**Pull request by ksuess**
Compile ++plone++static/plone-legacy-compiled.js with correct resource url of comments.js and others.
see #2626

### CODE REVIEW

**Comment by ksuess** [comment]
@thet, you know this resource registry very well. Can you have a look?

**Comment by jensens** [comment]
hm, whats the state here?

**Comment by ksuess** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/resources/browser/cook.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def cookWhenChangingSettings(context, bundle=None):
                     logger.warn('Could not find resource: %s' , css_resource)
         if not resource.js or not js_path:
             continue
-        js_url = siteUrl + '/' + resource.js
+        js_url = resource.js
```
Comment (thet) [inline Products/CMFPlone/resources/browser/cook.py]: what is the final url which is generated for js_url and css_url?
is it relative to the current context, e.g. when calling https://domain.com/subsite/context1/context2, is it https://domain.com/subsite/context1/context2/plone-logged-in.js or is it https://domain.com/subsite/plone-logged-in.js ?
If it's the first, that would be problematic as we cannot load the same resource for each context in the site separately.

Comment (ksuess) [reply Products/CMFPlone/resources/browser/cook.py]: There is no final url for this resource (comment.js, event.js,..). 
js_url is used just for cooking. A somehow final url is the one of the cooked js plone-legacy-compiled.js

**Comment by ksuess** [comment]
The issue is about **cooking** ++plone++static/plone-legacy-compiled.js

before:

    js_url = siteUrl + '/' + resource.js
    response = subrequest(js_url)

after:

    js_url = '/' + resource.js
    response = subrequest(js_url)


before means: subrequest is building path

    /VirtualHostBase/http/subdomain.example.com:80/VirtualHostRoot/_vh_somepath//somepath/Plone/++resource++plone.app.discussion.javascripts/comments.js
which is false and the resource cannot be found. So cook.py gets no response to include in cooked javascript ++plone++static/plone-legacy-compiled.js
cook.py prints the following in ++plone++static/plone-legacy-compiled.js:

    /* Could not find resource: https://subdomain.example.com/somepath/Plone/++resource++plone.app.discussion.javascripts/comments.js */

after means: subrequest is building path

    /VirtualHostBase/http/subdomain.example.com:80/VirtualHostRoot/_vh_somepath/Plone/++resource++plone.app.discussion.javascripts/comments.js
    which can be found and the response is included in cooked javascript ++plone++static/plone-legacy-compiled.js
cook.py prints the following in ++plone++static/plone-legacy-compiled.js:

    /* resource: ++resource++plone.app.discussion.javascripts/comments.js */
    if(require===undefined)require=function(reqs,torun){'use strict';return torun(window.jQuery);}; ...

**Pull request merged**